Hudl.Mjolnir/Command/Command.cs

```diff
@@ -181,12 +188,22 @@ internal Command(string group, string name, string breakerKey, string poolKey, T
 
             _log = LogManager.GetLogger("Hudl.Mjolnir.Command." + _name);
 
+            if (TimeoutsIgnored)
+            {
+                _log.Debug("Creating command with timeout disabled.");
+            }
             var timeout = GetTimeoutConfigurableValue(_name).Value;
             if (timeout <= 0)
             {
-                timeout = (long)defaultTimeout.TotalMilliseconds;
+                timeout = (long) defaultTimeout.TotalMilliseconds;
+            }
+            else
+            {
+                _log.DebugFormat("Timeout configuration override for this command of {0}",timeout);
+                //I think it is sensible to apply this timeout to the command if it has been explicitly set in configuration. 
+                //Therefore setting TimeoutsFullyDisabled to false regardless
+                TimeoutsIgnored = false;
```

Comment:
This feels a little hidden to me. As a user of the library, if I set `mjolnir.ignoreTimeouts=True`, that feels like a pretty assertive thing (like, I expect it to ignore *all* timeouts). If I were to set the config to True and still encounter a timeout, I'd be confused. Additionally, in local development, I'm almost positive that we have 0 people with per-command timeouts configured, so I'm not sure there's much of a problem to address by doing this.